marklogic-data-hub-central/ui/e2e/cypress/support/pages/detail.tsx

```diff
@@ -37,11 +37,11 @@ class DetailPage {
   }
 
   getDocumentJSON() {
-    return cy.get(".pretty-json-container");
+    return cy.findByTestId("json-container");
   }
 
   getDocumentXML() {
-    return cy.get("[data-cy=xml-document]");
+    return cy.findByTestId("xml-container");
```

Comment:
@timur-isangulov I see getDocumentJSON and getDocumentXML methods are using the same test id's in two different files, Any reason for not combining them together ?

Reply:
Not sure why there were a duplicates but I combined them. Thanks.